ethereumj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -91,8 +91,8 @@ public void execute() {
 
 
         // FIND OUT THE TRANSACTION TYPE
-        byte[] receiverAddress = null;
-        byte[] code = EMPTY_BYTE_ARRAY;
```

Comment:
no - it gets overwritten shortly afterwards in any case - assigning  EMPTY_BYTE_ARRAY would have no effect at all.